src/go/rpk/pkg/cli/cmd/cluster/partitions/status.go

```diff
@@ -0,0 +1,79 @@
+package partitions
+
+import (
+	"fmt"
+
+	"github.com/redpanda-data/redpanda/src/go/rpk/pkg/api/admin"
+	"github.com/redpanda-data/redpanda/src/go/rpk/pkg/config"
+	"github.com/redpanda-data/redpanda/src/go/rpk/pkg/out"
+	"github.com/spf13/afero"
+	"github.com/spf13/cobra"
+)
+
+func NewBalancerStatusCommand(fs afero.Fs) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "balancer-status",
+		Short: "Queries cluster for partition balancer status",
+		Long: `Queries cluster for partition balancer status:
+
+If continuous partition balancing is enabled, redpanda will continuously
```

Comment:
If this is the Redpanda product, then use "Redpanda". If it is a process, then use `redpanda`.

Reply:
Thanks, in this case, it is the process.